<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HATI-SDG00096" failures="0" tests="4" name="OPDTest.Cancel_Existing_appointment_against_Department" time="237.125" errors="0" timestamp="2024-07-29T11:43:50 IST" skipped="0">
  <testcase classname="OPDTest.Cancel_Existing_appointment_against_Department" name="TC_01_Verify_that_user_is_able_to_login_the_application" time="14.326"/>
  <testcase classname="OPDTest.Cancel_Existing_appointment_against_Department" name="TC_02_Verify_That_User_Is_Able_To_Do_Quick_Registration_to_book_the_existing_appointmet" time="67.803"/>
  <testcase classname="OPDTest.Cancel_Existing_appointment_against_Department" name="TC_03_Verify_that_user_is_able_to_book_the_Existing_appointment__against_Department_to_cancel_appointment" time="106.874"/>
  <testcase classname="OPDTest.Cancel_Existing_appointment_against_Department" name="TC_04_Verify_that_Existing_appointment_is_cancelled_succsessfull_against_Department" time="48.122"/>
</testsuite> <!-- OPDTest.Cancel_Existing_appointment_against_Department -->
